pkg/api/serve.go

```diff
@@ -37,6 +37,16 @@ const (
 	extensionValidationExcludeBody = "x-validation-exclude-body"
 )
 
+func MakeAuthMiddleware(logger logging.Logger, cfg *config.Config, middlewareAuthenticator auth.Authenticator, authService auth.Service) (func(next http.Handler) http.Handler, error) {
```

Comment:
Move to `pkg/api/auth_middleware.go` - add code doc to explain that this middleware will check all calls against our basic auth/cookie/jwt.

Reply:
Done